examples/wifi/wifi_aware/nan_console/main/nan_main.c

```diff
@@ -235,7 +235,10 @@ static int wifi_cmd_nan_disc(int argc, char **argv)
             nan_cfg.warm_up_sec = nan_args.warmup_time->ival[0];
         }
 
-        g_nan_netif = esp_netif_create_default_wifi_nan();
+        if(!g_nan_netif){
+            g_nan_netif = esp_netif_create_default_wifi_nan();    
+        }
+
         if ((esp_wifi_nan_start(&nan_cfg)) != ESP_OK) {
             ESP_LOGI(TAG, "Failed to start NAN");
             esp_netif_destroy_default_wifi(g_nan_netif);
```

Comment:
by extending the logic, `g_nan_netif` should be nullified here

Reply:
I didn't understand. Where exactly?